service/models/converters.py

```diff
@@ -72,27 +70,28 @@ def get_parameter_value(parameter):
     raise ValueError(f"Unsupported distribution type: {dist_type}")
 
 
-def get_static_intervention_value(static_inter, model_map):
+def resolve_intervention_value(
+    intervetion: HMIStaticIntervention | HMIDynamicIntervention, model_map
+):
     """Get static intervention value with distribution and percentage handling"""
 
     # If the intervention is not a type of parameter or value type of percentage, return the value directly
-    if static_inter.type != "parameter" or static_inter.value_type != "percentage":
-        return torch.tensor(float(static_inter.value))
+    if intervetion.type != "parameter" or intervetion.value_type != "percentage":
```

Comment:
I believe this should be 
`if intervetion.value_type != "percentage":`

If intervention `type` is `state` and `value_type` is `value` then should we not be returning the `intervention.value`?